### DISCUSSION TITLE
Show schema columns that are hidden

### DISCUSSION THREAD

**Discussion by tekumara** [Eventual-Inc/Daft#1866] [Q&A]
`df.schema()` shows some columns as hidden. How can I see the schema of these columns?

![hidden](https://private-user-images.githubusercontent.com/17691182/290469781-651b8516-7220-4b5e-a558-c321bd56c30d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDc2NTMyMTYsIm5iZiI6MTcwNzY1MjkxNiwicGF0aCI6Ii8xNzY5MTE4Mi8yOTA0Njk3ODEtNjUxYjg1MTYtNzIyMC00YjVlLWE1NTgtYzMyMWJkNTZjMzBkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjExVDEyMDE1NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAwNzMxOTg0MGY1MjYyYmJiYWMwMmM4Y2FlZTIyOTgzNzdiOTdmODdlNDI5ZDBlOGNlMzVkZmQwODkyMDllMmQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.c8J_EnOTDB2gptI_ZGfpJ3kfVTxUVeGyrGz0oOoaeCg)

**Comment by samster25** [comment]
Hi @tekumara, Thanks for making an issue! Our `repr` for schema is terminal adaptive so if you zoom out in your terminal (increasing the width), you'll see more columns of the schema. I will work on this [issue](https://github.com/Eventual-Inc/Daft/issues/1725) and target a fix for our next release!

**Comment by samster25** [comment]
@tekumara Just put out a [PR](https://github.com/Eventual-Inc/Daft/pull/1868)! Let me know what you think!

![image](https://github.com/Eventual-Inc/Daft/assets/2550285/f117a38c-4362-4134-8a7a-05545a443d83)

**Reply by tekumara** [reply]
Thank you that looks nice! Looking forward to trying it out.

**Reply by samster25** [reply]
`getdaft==0.2.14` now includes this behavior!
